src/frontend/metrics/generateMetricsReport.ts

```diff
@@ -0,0 +1,38 @@
+import type {ReadonlyDeep} from 'type-fest';
+import type {Observation} from '@mapeo/schema';
+import positionToCountries from './positionToCountries';
+
+export default function generateMetricsReport({
+  packageJson,
+  os,
+  osVersion,
+  screen,
+  observations,
+}: ReadonlyDeep<{
+  packageJson: {version: string};
+  os: string;
```

Comment:
would it be useful to limit the typeing of os? eg
``` ts
type Os = 'android' | 'ios' | 'win32' //... etc
```

Reply:
Done in c6be4982e9f41ee2327876ac24de7948d5cd7e53.